notebooks/basics/single-systems.ipynb

```diff
@@ -970,20 +976,26 @@
    "source": [
     "### 2.3 Unitary operations <a id='single-systems-unitary'></a>\n",
     "\n",
-    "Thus far it may not be evident why quantum information is fundamentally different from classical information.\n",
+    "Thus far, it may not be evident why quantum information is fundamentally different from classical information.\n",
     "That is, when a quantum state is measured, the probability to obtain each classical state is given by the absolute value squared of the corresponding vector entry — so why not simply record these probabilities in a probability\n",
     "vector?\n",
     "\n",
     "The answer, at least in part, is that the set of allowable *operations* that can be performed on a quantum state is different than it is for classical information.\n",
     "Similar to the probabilistic setting, operations on quantum states are linear mappings — but rather than being represented by stochastic matrices as in the classical case, operations on quantum state vectors are represented by *unitary* matrices.\n",
     "\n",
-    "A square matrix $U$ having complex number entries is unitary if it satisfies the equalities\n",
+    "A square matrix $U$ having complex number entries is unitary if it satisfies the equations\n",
     "\n",
     "$$\n",
-    "  U^{\\class{conjugate-transpose}{\\dagger}} U = \\class{identity-matrix}{\\mathbb{1}} = U U^{\\dagger}.\n",
+    "  \\begin{aligned}\n",
+    "    U U^{\\dagger} &= \\class{identity-matrix}{\\mathbb{1}} \\\\\n",
+    "    U^{\\class{conjugate-transpose}{\\dagger}} U &= \\class{identity-matrix}{\\mathbb{1}}.\n",
+    "  \\end{aligned}\n",
     "$$\n",
     "\n",
-    "Here, $U^{\\dagger}$ is the *conjugate transpose* of $U$, meaning the matrix obtained by transposing $U$ and taking the complex conjugate of each entry, and $\\mathbb{1}$ is the identity matrix.\n",
+    "Here, $\\mathbb{1}$ is the identity matrix, and $U^{\\dagger}$ is the *conjugate transpose* of $U$, meaning the matrix obtained by transposing $U$ and taking the complex conjugate of each entry.\n",
+    "$$\n",
+    "  U^{\\dagger} = (U^{T})^*\n",
```

Comment:
I don't want to use star for (entry-wise) complex conjugation — I prefer a bar over the matrix. In my worldview, a dagger is an antiquated star, and I've tended to use stars rather than daggers to denote the conjugate transpose.